src/pages/landing/projects/components/NoteCard.tsx

```diff
@@ -0,0 +1,110 @@
+import { useState } from 'react';
+import { Text, Box, Flex, Link, useBreakpointValue } from '@chakra-ui/react';
+
+const ImageWithReload = ({ src, alt, ...rest }) => {

```

Comment:
Image with Reload is already a component, that is present, you can simply import and use it, it has more functionality and handles the case of showing the grey image when image fails to load.
Please import and use the existing component.

I'm curious why you created a separate component here?